src/screens/OavMover.tsx

```diff
@@ -258,6 +265,149 @@ export function SideDrawer() {
+          </Grid2>
+          <Grid2 size={4}>
+            <Button
+              style={buttonStyle}
+              onClick={() =>
+                submitAndRunPlanImmediately("moveto", {
+                  place: "zero",
+                })
+              }
+            >
+              Go to Origin
+            </Button>
+          </Grid2>
+          <Grid2 size={4}>
+            <Button
+              style={buttonStyle}
+              onClick={() =>
+                submitAndRunPlanImmediately("moveto", {
+                  place: "f1",
+                })
+              }
+            >
+              Go to Fiducial 1
+            </Button>
+          </Grid2>
+          <Grid2 size={4}>
+            <Button
+              style={buttonStyle}
+              onClick={() =>
+                submitAndRunPlanImmediately("moveto", {
+                  place: "f2",
+                })
+              }
+            >
+              Go to Fiducial 2
+            </Button>
+          </Grid2>
+          <Grid2 size={4}>
+            <Button style={buttonStyle}>Set Fiducial 0</Button>
+          </Grid2>
+          <Grid2 size={4}>
+            <Button
+              style={buttonStyle}
+              onClick={() =>
+                submitAndRunPlanImmediately("fiducial", {
+                  point: "1",
+                })
+              }
+            >
+              Set Fiducial 1
+            </Button>
+          </Grid2>
+          <Grid2 size={4}>
+            <Button
+              style={buttonStyle}
+              onClick={() =>
+                submitAndRunPlanImmediately("fiducial", {
+                  point: "2",
+                })
+              }
+            >
+              Set Fiducial 2
+            </Button>
+          </Grid2>
+          <Grid2 size={6}>
+            <Button
+              style={buttonStyle}
+              onClick={() => submitAndRunPlanImmediately("cs_maker", {})}
+            >
+              Make Coordinate System
+            </Button>
+          </Grid2>
+          <Grid2 size={6}>
+            <Button
+              style={buttonStyle}
+              onClick={() => submitAndRunPlanImmediately("block_check", {})}
+            >
+              Block Check
+            </Button>
+          </Grid2>
+        </Grid2>
+      </Box>
+      <Dialog
+        onClose={handleClose}
+        aria-labelledby="customized-dialog-title"
+        open={open}
+      >
+        <DialogTitle sx={{ m: 0, p: 2 }} id="customized-dialog-title">
+          How to use Co-ordinate System Setup
+        </DialogTitle>
+        <IconButton
+          aria-label="close"
+          onClick={handleClose}
+          sx={(theme) => ({
+            position: "absolute",
+            right: 8,
+            top: 8,
+            color: theme.palette.grey[500],
+          })}
+        >
+          <Close />
+        </IconButton>
+        <DialogContent dividers>
+          <img src="Oxford Chip Diagram.excalidraw.svg" alt="" />
```

Comment:
I think it might be tidier to have all svg files in the same place, so would move this file to `src/assets` 
(We may want to remove the `public` directory actually - but that's a separate PR)